arrow-core-data/src/main/kotlin/arrow/core/computations/suspended/NullableBindSyntax.kt

```diff
@@ -0,0 +1,10 @@
+package arrow.core.computations.suspended
+
+import arrow.continuations.generic.DelimitedScope
+
```

Comment:
Binding syntax for a [nullable] within a continuation context.

Reply:
moved `NullableBindSyntax.kt` into `nullable` as a private class